src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -137,6 +161,15 @@ public static boolean deleteFolder(final File file)
         return false;
     }
 
+    public static boolean deleteFile(final File file)
```

Comment:
just remove it, it doesn't belong in this. Add it in the CoreProtect PR if you make any.